schema/admonitions.schema.json

```diff
@@ -39,14 +39,15 @@
                   { "$ref": "#/$defs/AdmonitionTitle" },
                   { "$ref": "myst.schema.json#/$defs/FlowContent" }
                 ]
-              }
+              },
+              "description": "An optional `admonitionTitle` followed by the admonitions content."
             },
             "position": {},
             "data": {}
           },
           "additionalProperties": false
         },
-        { "$ref": "unist.schema.json#/$defs/Node" }
+        { "$ref": "unist.schema.json#/$defs/Parent" }
```

Comment:
Should we swap this back to `Node` like you did for cross reference? We are already validating `children` above in a more specific way; could cause problems to validate again here if there's some conflict.

Reply:
Sounds good, switched back.